alfresco-transformer-base/src/main/resources/application.yaml

```diff
@@ -48,3 +48,12 @@ management:
 
 container:
   name: ${HOSTNAME:t-engine}
+
```

Comment:
You can use the property files in the specific projects: https://github.com/Alfresco/alfresco-transform-core/blob/master/alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/main/resources/application-default.yaml